src/main/scala/org/cs1331/gitsubmitter/GitSubmitterPlugin.scala

```diff
@@ -0,0 +1,30 @@
+package org.cs1331.gitsubmitter
+
+import java.time.LocalDateTime
+import sbt._
+import Keys._
+
+object GitSubmitterPlugin extends AutoPlugin {
+
+  override lazy val projectSettings = Seq(commands += submitCommand)
+
+  lazy val submitCommand = Command.command("submit") { (state: State) =>
+    val prismId = readLine("Enter your prism id: ")
+    val repoSuffix = s"-$prismId"
```

Comment:
Correct. I phrased that poorly.  So the repo name is "hw-hwname" by default, and if you set appendUsername to true, it will add "-jdoe3" to the end.